code/modules/admin/smites/headset_explosion.dm

```diff
@@ -0,0 +1,28 @@
+/// explode target headset dealing head brain and eye damage
+/datum/smite/headset_explosion
+	name = "Headset Explosion"
+
+/datum/smite/headset_explosion/effect(client/user, mob/living/carbon/human/target)
```

Comment:
is this a tg port?

Reply:
This one is an original, inspired by the time everybody received an exploding headset for a round. As such all shitcode in this smite is 100% on me.

Reply:
you shouldnt put the new smite with the smite ports split it out please